pkg/psalabelsyncer/sccrolecache_test.go

```diff
@@ -1003,6 +1004,91 @@ func Test_saToSCCCache_handleSCCDeleted(t *testing.T) {
 	}
 }
 
+func TestSCCRoleCache_BySAIndexKeys(t *testing.T) {
+	newRBObj := func(rbNS string, subjects ...string) interface{} {
+		obj := &rbacv1.RoleBinding{
+			ObjectMeta: metav1.ObjectMeta{Namespace: rbNS},
+			Subjects:   []rbacv1.Subject{},
+		}
+
+		for i := 0; i < len(subjects); i += 2 {
```

Comment:
not a big fan of this `key1, key2` approach, I'd rather these have these explicitly listed